src/specs/data_structures.md

```diff
@@ -995,3 +995,14 @@ Various [consensus parameters](consensus.md#system-parameters) are committed to
 | `availableDataOriginalSquareMax` | `uint64`                              | The `AVAILABLE_DATA_ORIGINAL_SQUARE_MAX`. |
 
 In order to compute the `consensusHash` field in the [block header](#header), the above list of parameters is [hashed](#hashing).
+
+## StateFraudProof
+
+| name                       | type                                                                 | description                                                            |
+|----------------------------|----------------------------------------------------------------------|------------------------------------------------------------------------|
+| `headers`                  | [Header](#header)`[]`                                                | Headers of two consecutive blocks containing `availableDataRoot` of type [HashDigest](#hashdigest) committing to transaction and intermediate state root data                        |
```

Comment:
My intuition is that is that this should be two fields then instead of a slice/vector of headers.

Reply:
I agree, I will change accordingly.

Reply:
Done.